cmd/prysmctl/withdrawal/cmd.go

```diff
@@ -0,0 +1,56 @@
+package withdrawal
+
+import (
+	"os"
+
+	"github.com/prysmaticlabs/prysm/v3/cmd"
+	"github.com/prysmaticlabs/prysm/v3/runtime/tos"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	BeaconHostFlag = &cli.StringFlag{
+		Name:  "beacon-node-host",
+		Usage: "host:port for beacon node to query",
+		Value: "127.0.0.1:3500",
+	}
+	PathFlag = &cli.StringFlag{
+		Name:    "path",
+		Aliases: []string{"p"},
+		Usage:   "path to the signed withdrawal messages JSON",
+	}
+	SkipPromptsFlag = &cli.BoolFlag{
+		Name:  "skip-prompts",
+		Usage: "DANGER:User accepts responsibility of all input data without prompt verification! skips all prompts for verifying each provided signed withdrawal message to for setting the withdrawal address",
+	}
+)
+
+var Commands = []*cli.Command{
+	{
+		Name:    "set-withdrawal-addresses",
+		Aliases: []string{"swa"},
+		Usage:   "command for setting the withdrawal ethereum addresses to their associated validator keys. WARN: once set values are included they can no longer be updated.",
```

Comment:
`WARN` looks odd, maybe just say `WARNING`?

Reply:
another nitpick: `command for setting` is kinda redundant since we know we're in a command. Suggest more direct language like `Assign Ethereum withdrawal addresses to validator keys`.